### ISSUE TITLE
isIos fails on iPad 13

### ISSUE THREAD

**Issue by Noitidart** [braintree/browser-detection#45]
On iPad 13, by default, it's requesting desktop websites. So `isIos()` always returns false. We might need a workaround like this to detect `isIos` - https://stackoverflow.com/questions/58019463/how-to-detect-device-name-in-safari-on-ios-13-while-it-doesnt-show-the-correct

```
let isIOS = /iPad|iPhone|iPod/.test(navigator.platform) ||
(navigator.platform === 'MacIntel' && navigator.maxTouchPoints > 1)

The first condition is old-fashioned and works with previous versions, while the second condition works for iPad OS 13 which now identifies itself as:

    "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15) AppleWebKit/605.1.15 (KHTML, like Gecko)"

which by all platform detectors I know is not detected (for now) neither as mobile nor desktop.

So since iPad OS now calls itself Macintosh, but real macs have no multi-touch support, this solution is ideal to detect iPad OS devices which are the only multi-touch "Macintosh" devices in existence.
```

**Comment by SancSalix** [comment]
We'll open a ticket to make a distinction between those, thank you for letting us know!

**Comment by jplukarski** [comment]
This was released in v1.16.0

**Comment by Noitidart** [comment]
Thanks very much! I can remove my url now.
